lib/minefield.rb

```diff
@@ -0,0 +1,198 @@
+require_relative 'cell'
```

Comment:
Never knew about require_relative, cool stuff! Not sure if I'd use it in this scenario, though.